apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/TimedBufferingConsumer.cs

```diff
@@ -43,10 +43,9 @@ internal TimedBufferingConsumer(IConsumer<T> consumer, TimeSpan waitTime, IThrea
                 {
                     Flush();
                 }
-                catch (Exception)
+                catch (Exception ex)
                 {
-                    // TODO(atarafamas): This is a short term solution to ensure 
-                    // we do not kill a process. See issue #2182 (currently in backlog) to track the long solution.
+                    timerExceptionHandler?.Invoke(ex);
```

Comment:
Possibly keep a comment for why we're swallowing the exception?

Reply:
Yep done.